azlinux-jupyter-scipy/Dockerfile

```diff
@@ -0,0 +1,310 @@
+ARG ROOT_CONTAINER=quay.io/cdis/amazonlinux:latest
```

Comment:
could you add a comment at the top of this file, or a README in `azlinux-jupyter-scipy/`, to explain which original dockerfiles this one is based on and that the base image was updated to amazonlinux

Reply:
Added a Readme file [here](https://github.com/uc-cdis/containers/blob/611ef45ec744d3defe19d6cccbac9404964e4e98/azlinux-jupyter-scipy/README.md)